CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Changelog for NeoFS Node
 ### Updated
 - Update minimal supported Go version up to v1.19 (#2485)
 - Update `neo-go` to `v0.102.0` (#2221, #2309, #2596)
+- `neofs-lens` `inspect` object commands to `get` with `inspect` deprecation (#2548)
```

Comment:
@AliceInHunterland, @roman-khimov, are we sure about section choice?

Reply:
Suggestions? Can be fixed if needed.

Reply:
i always thought that `Updated` is about `go.mod` file (external dependencies in short) and our changes should be added to the `Updating from` section, no?